src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/EnumShouldNotHaveDuplicatedValues.cs

```diff
@@ -73,7 +73,8 @@ public sealed override void Initialize(AnalysisContext context)
                     var fieldInitializerOperation = (IFieldInitializerOperation)oc.Operation;
 
                     if (fieldInitializerOperation.InitializedFields.Length != 1 ||
-                        !fieldInitializerOperation.Value.ConstantValue.HasValue)
+                        !fieldInitializerOperation.Value.ConstantValue.HasValue ||
+                        fieldInitializerOperation.Value.ConstantValue.Value == null)
```

Comment:
I am wondering if it's possible to be true here and then false when we reach the line 96. If so, we might want to move (or duplicate) those 2 checks just before the`AddOrUpdate`.

Reply:
@Evangelink isn’t it the same field initializer operation used at both places? When do you think the constant value can change?